### DISCUSSION TITLE
Debugging in Eclipse

### DISCUSSION THREAD

**Discussion by davidwowa** [DependencyTrack/dependency-track#2915] [Q&A]
Hello, maybe exists one description, how is possible to start and debug dependency-track in eclipse ? Thanks

**Comment by nscuro** [comment]
I haven't used Eclipse in years, but the way we do it with IntelliJ is to run this Maven command:

```shell
mvn jetty:run -P enhance -Dlogback.configurationFile=src/main/docker/logback.xml
```

And then attach the Java debugger to that Maven process. In IntelliJ, the command execution and debugger attachment is seamless. I am sure you can do something similar in Eclipse, IIRC it had a great Maven integration.

See https://github.com/DependencyTrack/dependency-track/blob/master/DEVELOPING.md#debugging for more details.
